### PR TITLE
Ensure database is properly closed when shutting down the server

### PR DESCRIPTION
**Pull request by jgiannuzzi**

### CODE REVIEW

**Review by suprjinx — DISMISSED** [review]

File: tests/integration/golang/mlflow/experiment/delete_test.go [hunk 1]
```diff
@@ -40,6 +40,10 @@ func (s *DeleteExperimentTestSuite) SetupTest() {
 	s.fixtures = fixtures
 }
 
+func (s *DeleteExperimentTestSuite) TearDownTest() {
```
Comment (dsuhinin) [inline tests/integration/golang/mlflow/experiment/delete_test.go:43]: this is not really important in tests.

Comment (dsuhinin) [reply tests/integration/golang/mlflow/experiment/delete_test.go:43]: this is like with table names today =)

File: tests/integration/golang/mlflow/run/log_batch_test.go [hunk 2]
```diff
@@ -65,6 +65,12 @@ func (s *LogBatchTestSuite) SetupTest() {
 	s.run = run
 }
 
+func (s *LogBatchTestSuite) TearDownTest() {
+	assert.Nil(s.T(), s.runFixtures.CloseDB())
+	assert.Nil(s.T(), s.metricFixtures.CloseDB())
+	assert.Nil(s.T(), s.experimentFixtures.CloseDB())
+}
+
```
Comment (dsuhinin) [inline tests/integration/golang/mlflow/run/log_batch_test.go:73]: emmm, we definitely don't need to take case about that.

File: tests/integration/golang/mlflow/experiment/delete_test.go (hunk 1 above)
Comment (jgiannuzzi) [reply tests/integration/golang/mlflow/experiment/delete_test.go]: alright, I have removed all the test bits!

**Review by dsuhinin — APPROVED** [review]

**Pull request merged**